examples/bert/bert_with_hypertuning_main.py

```diff
@@ -104,7 +96,9 @@ def forward(self,  # type: ignore
         for more details.
 
         Args:
-            `batch`: :class:`texar.data.Batch`
+            `batch`: :class:`texar.data.Batch`. (See
+            https://texar-pytorch.readthedocs.io/en/latest/code/data.html#texar.torch.data.Batch
```

Comment:
This and the following line should be indented.

Reply:
Resolved in aa75023